shop/src/main/java/ro/msg/learning/shop/Controllers/OrderController.java

```diff
@@ -0,0 +1,40 @@
+package ro.msg.learning.shop.Controllers;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+import ro.msg.learning.shop.DTOs.orderDto.OrderDTOInput;
+import ro.msg.learning.shop.DTOs.orderDto.OrderDTOOutput;
+import ro.msg.learning.shop.Services.OrderServiceImpl;
+import ro.msg.learning.shop.exceptions.OrderPlacingException;
+
+import java.util.List;
+
+@Controller
```

Comment:
you can use the newer @RestController and respectively @GetMapping or @PostMapping annotation which combines the behavior of  @Controller and @ResponseBody together.